### ISSUE TITLE
Image alt text should be translated

### ISSUE THREAD

**Issue by iangmhill** [MAPC/digitalhub#95]
**Describe the feature. What should a user be able to do?**
Automated translation covers events and pages but no alt text on images.

**Why do we need this feature?**
A non-english-speaking screen reader user might be confused by english alt text on images despite the page being translated.

**Comment by mzagaja** [comment]
Superseded by #385
